src/main/java/ch/njol/skript/expressions/ExprTamer.java

```diff
@@ -52,10 +50,9 @@
 	@Override
 	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
 		if (!ScriptLoader.isCurrentEvent(EntityTameEvent.class)) {
-			Skript.error("tamer expression can be only used in entity tame event.");
+			Skript.error("the expression `tamer` may only be used in the entity tame event.");

```

Comment:
Looks like the quotes you use are not ASCII ones (character '). A mistake, or is there some reason for this?

Changes to language are good.